site/src/sphinx/mirroring.rst

```diff
@@ -208,6 +214,10 @@ repositories defined in ``/mirrors.json``:
   - the passphrase of ``privateKey`` if the private key is encrypted.
     If unspecified or ``null``, the private key should not be encrypted.
 
+- ``accessToken`` (string)
+
+  - the access token which is used for access token-based authentication such as GitHub Personal Access Token.
```

Comment:
Let's link https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token to `GitHub Personal Access Token`

Reply:
OK. I'll do it!

Reply:
@ikhoon I added at [4afa19b](https://github.com/line/centraldogma/pull/813/commits/4afa19be8184425bc3b237b60be495ab267668c7)